Doc/library/http.server.rst

```diff
@@ -392,7 +392,7 @@ provides three different variants:
       contents of the file are output. If the file's MIME type starts with
       ``text/`` the file is opened in text mode; otherwise binary mode is used.
 
-      For example usage, see the implementation of the :func:`test` function
+      For example usage, see the implementation of the :source:`~http.server.test` function
```

Comment:
This is not correct, you need a file path like `Lib/http/server.py`. See the top of https://docs.python.org/3.10/library/dbm.html for an example of how it is rendered.

Reply:
Right, thanks for your help :)